src/tools/clippy/tests/ui/integer_arithmetic.stderr

```diff
@@ -123,6 +123,18 @@ error: integer arithmetic detected
 LL |     i >>= 2;
    |     ^^^^^^^
 
+error: integer arithmetic detected
+  --> $DIR/integer_arithmetic.rs:68:28
+   |
+LL |         const ASSOC: i32 = 1 + 1;
```

Comment:
Why does this change here and in the other similar PR. This has something to do with associated constants.